src/main/resources/db/migration/V10__soknad_slett_vedlegg.sql

```diff
@@ -1 +1,2 @@
-ALTER TABLE soknad DROP COLUMN vedlegg;
+ALTER TABLE soknad
```

Comment:
{"@timestamp":"2020-08-24T09:09:10.431+02:00","@version":"1","message":"Application run failed","logger_name":"org.springframework.boot.SpringApplication","thread_name":"main","level":"ERROR","level_value":40000,"stack_trace":"org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flywayInitializer' defined in class path resource [org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration$FlywayConfiguration.class]: Invocation of init method failed; nested exception is org.flywaydb.core.api.FlywayException: Validate failed: \nMigration checksum mismatch for migration version 8\n-> Applied to database : 129735906\n-> Resolved locally    : 2011507364\nMigration checksum mismatch for migration version 10\n-> Applied to database : -332795986\n-> Resolved locally    : 786145427\n\n\tat org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1794)\n\tat org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594